server/controller.js

```diff
@@ -77,4 +77,10 @@ module.exports.element = (req, res) => {
       }
     });
 
+};
+
+module.exports.bundle = (req, res) => {
+
+  res.sendFile(path.join(__dirname, '..', 'client', 'dist', 'course-content.js'));
+
```

Comment:
is that still being used?

Reply:
Yes, I'm using the '/bundle' endpoint to send my file rather than using express.static